### PR TITLE
add temporary lightest blank favicon, fix #422

### LINKED ISSUE #422: Avoir une favicon
En moindre effort, mettre une [image dummy](https://github.com/cafebabel/cafebabel.com/blob/master/cafebabel/templates/500.html#L5) pour éviter une requête inutile vers le serveur.

### PR DESCRIPTION
**Pull request by johangiraud**
fix #422

### CODE REVIEW

File: cafebabel/templates/base.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <meta charset=utf-8>
 <meta name=viewport content="width=device-width, minimum-scale=1, initial-scale=1, shrink-to-fit=no">
 <title>{% block meta_title %}pan-European news, features and opinions{% endblock %} – Cafebabel</title>
+<link rel=icon href="data:;base64,iVBORw0KGgo=">
```
Comment (davidbgk) [inline cafebabel/templates/base.html]: Maybe adding a comment about why that icon (for now?)

Comment (johangiraud) [reply cafebabel/templates/base.html]: yep ! https://github.com/cafebabel/cafebabel.com/pull/436/commits/9f5ea3dfd04e6e387ff643c9e089146c1f1db80c

**Comment by davidbgk** [comment]
https://realfavicongenerator.net/

**Comment by johangiraud** [comment]
Thanks @davidbgk #492

**Pull request merged**